<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeguimientosForm</class>
 <widget class="QWidget" name="SeguimientosForm">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="formLayout">
     <item row="0" column="0"><widget class="QLabel" name="labelCodigo"><property name="text"><string>Código</string></property></widget></item>
     <item row="0" column="1"><widget class="QLineEdit" name="inputCodigo"/></item>
     <item row="1" column="0"><widget class="QLabel" name="labelFecha"><property name="text"><string>Fecha</string></property></widget></item>
     <item row="1" column="1"><widget class="QLineEdit" name="inputFecha"/></item>
     <item row="2" column="0"><widget class="QLabel" name="labelEstado"><property name="text"><string>Estado</string></property></widget></item>
     <item row="2" column="1"><widget class="QLineEdit" name="inputEstado"/></item>
     <item row="3" column="0"><widget class="QLabel" name="labelCliente"><property name="text"><string>Cliente</string></property></widget></item>
     <item row="3" column="1"><widget class="QLineEdit" name="inputCliente"/></item>
     <item row="4" column="0"><widget class="QLabel" name="labelObra"><property name="text"><string>Obra</string></property></widget></item>
     <item row="4" column="1"><widget class="QLineEdit" name="inputObra"/></item>
     <item row="5" column="0"><widget class="QLabel" name="labelFechaEntrega"><property name="text"><string>Fecha Entrega</string></property></widget></item>
     <item row="5" column="1"><widget class="QLineEdit" name="inputFechaEntrega"/></item>
     <item row="6" column="0"><widget class="QLabel" name="labelDias"><property name="text"><string>Días</string></property></widget></item>
     <item row="6" column="1"><widget class="QLineEdit" name="inputDias"/></item>
     <item row="7" column="0"><widget class="QLabel" name="labelDesde"><property name="text"><string>Desde</string></property></widget></item>
     <item row="7" column="1"><widget class="QLineEdit" name="inputDesde"/></item>
     <item row="8" column="0"><widget class="QLabel" name="labelHasta"><property name="text"><string>Hasta</string></property></widget></item>
     <item row="8" column="1"><widget class="QLineEdit" name="inputHasta"/></item>
     <item row="9" column="0"><widget class="QLabel" name="labelGuiaID"><property name="text"><string>Guía ID</string></property></widget></item>
     <item row="9" column="1"><widget class="QLineEdit" name="inputGuiaID"/></item>
     <item row="10" column="0"><widget class="QLabel" name="labelFacturaID"><property name="text"><string>Factura ID</string></property></widget></item>
     <item row="10" column="1"><widget class="QLineEdit" name="inputFacturaID"/></item>
     <item row="11" column="0"><widget class="QLabel" name="labelNota"><property name="text"><string>Nota</string></property></widget></item>
     <item row="11" column="1"><widget class="QTextEdit" name="inputNota"/></item>
     <item row="12" column="0"><widget class="QLabel" name="labelBuscar"><property name="text"><string>Buscar</string></property></widget></item>
     <item row="12" column="1"><widget class="QLineEdit" name="inputBuscar"/></item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="botonesLayout">
     <item><widget class="QPushButton" name="btnAgregar"><property name="text"><string>Agregar</string></property></widget></item>
     <item><widget class="QPushButton" name="btnEditar"><property name="text"><string>Editar</string></property></widget></item>
     <item><widget class="QPushButton" name="btnEliminar"><property name="text"><string>Eliminar</string></property></widget></item>
     <item><widget class="QPushButton" name="btnBuscar"><property name="text"><string>Buscar</string></property></widget></item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tablaSeguimientos"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
